pom.xml

```diff
@@ -324,11 +300,13 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.19.1</version>
+                <version>${surefire.version}</version>
                 <configuration>
                     <excludes>
-                        <!-- TEMPORARY since NCE tests sometimes fail to load classes -->
-                        <exclude>jmri.jmrix.nce.**</exclude>
+                        <!-- ignore until #3340 is merged -->
+                        <exclude>jmri.jmrix.nce.**.Nce*</exclude>
+                        <!-- ignore until #3492 is resolved -->
+                        <exclude>jmri.jmrix.loconet.locormi.LnMessage*Test</exclude>
```

Comment:
This PR should NOT have been merged.  It still excludes tests from the locormi package.